sdk/attestation/azure-security-attestation/tests/test_attestation.py

```diff
@@ -132,13 +132,14 @@
 )
 
 
-class AttestationTest(AzureTestCase):
+class TestAttestation(AzureRecordedTestCase):
 
     # The caching infrastructure won't cache .well-known/openid_metadata responses so
     # mark the metadata related tests as live-only.
```

Comment:
It might be interesting to see if the test proxy actually does record .well-known/openid_metadata and thus this "live-only" restriction could be removed.

Reply:
Hi Larry, I believe we need to keep this live test Only restriction because when trying to run this recorded test, we get following type of error.

ValueError: ('Could not deserialize key data. The data may be in an incorrect format, it may be encrypted with an unsupported algorithm, or it may be an unsupported key type (e.g. EC curves with explicit parameters).', [_OpenSSLErrorWithText(code=503841036, lib=60, reason=524556, reason_text=b'error:1E08010C:DECODER routines::unsupported'), _OpenSSLErrorWithText(code=109052059, lib=13, reason=155, reason_text=b'error:0680009B:asn1 encoding routines::too long'), _OpenSSLErrorWithText(code=109052006, lib=13, reason=102, reason_text=b'error:06800066:asn1 encoding routines::bad object header'), _OpenSSLErrorWithText(code=109576458, lib=13, reason=524554, reason_text=b'error:0688010A:asn1 encoding routines::nested asn1 error')])